libraries/SITL/SITL.h

```diff
@@ -152,6 +152,7 @@ class SITL {
     AP_Float engine_mul;  // engine multiplier
     AP_Int8  engine_fail; // engine servo to fail (0-7)
     AP_Int8  gps_disable; // disable simulated GPS
+    AP_Int8  gps2_disable; // disable simulated GPS
```

Comment:
what is the advantage of a gps2_disable over the existing SIM_GPS2_ENABLE parameter?

Reply:
It's simulating NO_FIX state. Instead of NO_GPS. Actually SIM_GPS_DISABLE is a NO_FIX not a complete disable.

Reply:
That's why I've suggested renaming both. They can be renamed to SIM_GPS_NOLOCK & SIM_GPS2_NOLOCK.

It helps simulating the different switching options. From 1 to 2 and vice versa.

Reply:
What do you think?